### PR TITLE
Added Dockerfile template

### PR DESCRIPTION
**Pull request by volkm**
The Dockerfile can be used as a template for custom libraries/scripts/artifacts/etc.

### CODE REVIEW

File: doc/scripts/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+WORKDIR /opt/storm/build
+# If needed, checkout a specific branch
+#RUN git checkout specific_branch
+# Configure Storm
+RUN cmake .. "-DCMAKE_BUILD_TYPE=Release -DSTORM_DEVELOPER=OFF -DSTORM_LOG_DISABLE_DEBUG=ON -DSTORM_PORTABLE=ON"
+# Build binaries of Storm using a single thread
+RUN make binaries -j 1
+ENV PATH="/opt/storm/build/bin:$PATH"
+
+
```
Comment (sjunges) [inline doc/scripts/Dockerfile:32]: I would remove the build additional libraries; generally I do that in a separate docker file etc. We can add a docker file to the starter-template gits that does that part?

**Comment by sjunges** [comment]
Generally great; I just have two design decisions I do not fully understand. Happy to discuss offline and summarize our result here.

File: doc/scripts/Dockerfile (hunk 1 above)
Comment (volkm) [reply doc/scripts/Dockerfile:32]: Sounds good. Then this docker file is just for building the base Storm version.

**Comment by volkm** [comment]
Following our discussion, I restructured the Dockerfile and moved it to the root folder of Storm. The Dockerfile now enables the compilation of everything inside the current repo. Additional libraries must be specified in their own Dockerfile.
@sjunges Maybe check whether `.dockerignore` is correct.

**Comment by sjunges** [comment]
LGTM!

**Pull request merged**